lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -18,7 +18,7 @@ const pageFunctions = require('../../lib/page-functions.js');
  * @return {Promise<LH.Artifacts.Accessibility>}
  */
 /* c8 ignore start */
-async function runA11yChecks() {
+async function runA11yChecks(isTestMode = false) {
```

Comment:
i'm fine with naming bikeshedding. I didn't name it `isUnderTest` as we have that already and it doesnt have the same semantics.

Reply:
alternative would be to make a new paage function `runAxe` that returns the raw results, and have `runA11yChecks` call that and do the mapping to gatherer result. then the test can just call `runAxe`.

Reply:
this diverges a fair bit from the actual impl :/ WDYT about splitting the function into two `runAxe(resultTypes)` and `runA11yChecks` where we can just test `runAxe` directly?

Reply:
lol @connorjclark great minds!